python/sqlflow_submitter/pai/utils.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 The SQLFlow Authors. All rights reserved.
```

Comment:
I would suggest not to name source dirs or files by 'util'; a more precise name is often preferred.  How about renaming this file to be `oss_temporary.py`?

Reply:
I meant to provide some other functions in this file, anyway, I'll come up with a better name.